app/backend/src/clj/teet/contract/contract_queries.clj

```diff
@@ -0,0 +1,25 @@
+(ns teet.contract.contract-queries
+  (:require [teet.db-api.core :refer [defquery]]
+            [teet.contract.contract-db :as contract-db]
+            [teet.util.datomic :as du]
+            [teet.util.collection :as cu]))
+
+
+(defquery :contract/contract-page
+  {:doc "Return the single contracts information"
+   :context {db :db user :user}
+   :args {contract-ids :contract-ids}
+   :project-id nil
+   :authorization {}}
+  (let [[contract-id contract-part-id] contract-ids
+        contract-eid [:thk.contract/procurement-id+procurement-part-id [contract-id contract-part-id]]
+        result (-> (contract-db/get-contract
+              db
+              contract-eid)
+            (assoc
+              :thk.contract/targets
+              (contract-db/get-contract-target-information db contract-eid))
+            (cu/update-in-if-exists [:thk.contract/cost] str)
```

Comment:
there's two ways these are converted, to frontend (str) and back to backend (bigdec), should they be colocated in the model ns?